lib/src/body.dart

```diff
@@ -52,6 +52,13 @@ class Body {
         contentLength = encoded.length;
         stream = Stream.fromIterable([encoded]);
       }
+    } else if (body is List<int>) {
+      // Any data that is written to a Dart socket that is not a Uint8List or
+      // Int8List must be copied into a new list. Allow users to provide a
+      // typed data object to avoid this copy, which is important for performance
+      // of large file requests.
```

Comment:
This comment feels too detailed to me - the extra detail isn't actionable to future maintainers. It's great context to have in the commit message.

I'd go with 

```dart
// Avoid performance overheads from an unnecessary cast.
```

Reply:
Done